### PR TITLE
updating to latest deblur version

### PR DESCRIPTION
**Pull request by josenavas**
It adds the skip-trimming parameter

### CODE REVIEW

**Comment by josenavas** [comment]
This is failing because it is installing the wrong version of deblur (it is pulling version 0.1.7 from pypi rather than using the dependency links version).

Any idea on how to fix it?

**Comment by antgonza** [comment]
pip install https://github.com/biocore/deblur/archive/master.zip ?

**Comment by josenavas** [comment]
That would work for travis, but we will need to pay attention when updating/installing in systems, since it will not be pulling down the correct version of deblur.

File: qp_deblur/tests/test_deblur.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def test_deblur(self):
         fd, fp = mkstemp(suffix='_seqs.fastq')
```
Comment (antgonza) [inline qp_deblur/tests/test_deblur.py]: should this be demux? Also, do we still need the fastq?

**Pull request merged**